### PR TITLE
Add script to bump build numbers.

### PR DESCRIPTION
**Pull request by charlesmchen**
It's not the most polished script in the world, but it's safe and it'll fail with messages that will be useful for resolving any issue that comes up.

PTAL @michaelkirk

### CODE REVIEW

**Comment by charlesmchen** [comment]
Sample output on success:
```
matthews-iMac:Signal-iOS-4 matthew$ ./Scripts/bump_build_number.py
old_build_number: 2.13.0.18
new_build_number: 2.13.0.19
```
Sample output on failure:
```
matthews-iMac:Signal-iOS-4 matthew$ ./Scripts/bump_build_number.py
AM Scripts/bump_build_number.py
M  Signal/Signal-Info.plist

Git repository has untracked files.
```

File: Scripts/bump_build_number.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    build_number = str(1 + int(build_number))
+    new_build_number = old_build_number[:build_number_match.start(1)] + build_number
+    print 'new_build_number:', new_build_number
+    
+    text = text[:file_match.start(1)] + new_build_number + text[file_match.end(1):]
+    with open(plist_path, 'wt') as f:
+        f.write(text)
+        
+    output = subprocess.check_output(['git', 'add', '.'])
+    output = subprocess.check_output(['git', 'commit', '-m', 'Bump build from to %s.\n\n// FREEBIE' % new_build_number])
```
Comment (michaelkirk) [inline Scripts/bump_build_number.py]: "Bump build from **%s** to %s" ?

or just 

"Bump build to %s" ?

Comment (charlesmchen) [inline Scripts/bump_build_number.py]: I prefer the latter but let me know if you want to flip it.

**Comment by charlesmchen** [comment]
> Out of the box they do an integer (non-decimal) build number, which is also fine with me.

Let's stick with this for now, since we use the long version numbers in various places and I can't be bothered to change them.

**Pull request merged**